.travis.yml

```diff
@@ -38,3 +38,6 @@ script:
         source ./.CI/build_all;
       fi
 
+after_failure:
+  - "ls -l /Users/travis/miniconda/conda-bld/work/pybufr-ecmwf-0.82/build/lib.macosx-10.6-x86_64-2.7/pybufr_ecmwf/" 
+  - "cat /Users/travis/miniconda/conda-bld/work/pybufr-ecmwf-0.82/build/lib.macosx-10.6-x86_64-2.7/pybufr_ecmwf/f2py_build.log"
```

Comment:
This is just for debugging the osx build failure. Will be removed as soon as it is working.